include/mbgl/ios/MGLMapCamera.h

```diff
@@ -0,0 +1,36 @@
+#import "Mapbox.h"
+
+#pragma once
```

Comment:
Curious @1ec5 why this is required, for an iOS header which should be brought in with the once-safe `#import` vs. `#include` anyway.